src/quacc/recipes/onetep/_base.py

```diff
@@ -0,0 +1,77 @@
+"""Base jobs for Onetep."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase import Atoms
+
+### Do we need to import from quacc at this point we can do it directly from ase
+from quacc.calculators.onetep.onetep import Onetep, OnetepProfile
+from ase.calculators.onetep import OnetepTemplate
+from quacc.runners.ase import run_calc
+from quacc.schemas.ase import summarize_run
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from quacc.schemas._aliases.ase import RunSchema
+
+
+def base_fn(
+    atoms: Atoms = Atoms(),
```

Comment:
Please remove the optional keyword argument of `Atoms()`. This was a hack used in the Espresso calculator.